cmd/server/cmd/server.go

```diff
@@ -89,6 +90,33 @@ type serverArgs struct {
 	globalConfigFile string
 }
 
+func (sa *serverArgs) String() string {
+	var b bytes.Buffer
```

Comment:
Can we just reflect on the fields and do : b.WriteString(fmt.Sprint("'%s': '%v'\n", field.Name, field.Value)) ? There is a change this list might get out of sync.. (code review is the only way to prevent that from happening. Since the code review differ only shows the changed lines and it's immediate surrounding, chances are this will get missed in PR review).

Or just use [spew.dump]("github.com/davecgh/go-spew/spew"
) or %v on serverArgs.

Reply:
Thats what I tried first, but all fields are private at present, so , they will all have to be turned into public fields.

Reply:
Consider using spew.dump. I think that works on private fields too. We can just delete tall this code.